src/utils/symbolGenerators.ts

```diff
@@ -0,0 +1,65 @@
+///<reference path="../reference.ts" />
+
+module Plottable {
+
+  /**
+   * A SymbolGenerator is a function that takes in a datum and the index of the datum to
+   * produce an svg path string analogous to the datum/index pair.
+   *
+   * Note that SymbolGenerators used in Plottable will be assumed to work within a 100x100 square
+   * to be scaled appropriately for use within Plottable
+   */
+  export type SymbolGenerator = (datum: any, index: number) => string;
+
+  export module SymbolGenerators {
+
+    /**
+     * The radius that symbol generators will be assumed to have for their symbols.
+     */
+    export var SYMBOL_GENERATOR_RADIUS = 50;
+
+    export type StringAccessor = ((datum: any, index: number) => string);
+
+    /**
+     * A wrapper for D3's symbol generator as documented here:
+     * https://github.com/mbostock/d3/wiki/SVG-Shapes#symbol
+     *
+     * Note that since D3 symbols compute the path strings by knowing how much area it can take up instead of
+     * knowing its dimensions, the total area expected may be off by some constant factor.
+     *
+     * @param {string | ((datum: any, index: number) => string)} symbolType Accessor for the d3 symbol type
+     * @returns {SymbolGenerator} the symbol generator for a D3 symbol
+     */
+    export function d3Symbol(symbolType: string | StringAccessor) {
+      var typeToSize = (symbolTypeString: string) => {
+        var sizeFactor: number;
+        switch(symbolTypeString) {
+          case "circle":
+            sizeFactor = Math.PI;
+            break;
+          case "square":
+            sizeFactor = 4;
+            break;
+          case "cross":
+            sizeFactor = 20/9;
+            break;
+          case "diamond":
+            sizeFactor = 2 * Math.tan(Math.PI / 6);
+            break;
+          case "triangle-up":
+          case "triangle-down":
+            sizeFactor = Math.sqrt(3);
```

Comment:
They shouldn't need to go away.  Now that these numbers have been calculated, we shouldn't have any bounding issues from D3